### PR TITLE
Meeting Notes 2022-01-13 SARIF TC no. 50

### PR DESCRIPTION
**Pull request by sthagen**
This is the live pre-meeting version of the minutes draft for the 2022-01-13 meeting:

* [x] Current link for minutes approval of previous meeting (2021-12-09)
* [x] Current agenda link needs update when agenda draft link is available
* [x] Update table of participants during the roll call and the call for late joiners
* [x] Replace TODOs with actual minutes during the meeting
* [x] Update the decisions taken at the end of the meeting
* [x] Update the actions taken dito
* [x] Amend with post meeting draft revision
* [x] Ask for review and merge

### CODE REVIEW

**Comment by sthagen** [comment]
@dmk42, @eddynaka, @michaelcfanning, and all other participants: Please kindly review the current draft. Thanks.

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+* David: submitted the request to OASIS for the 15-day public review of the Errata on 2021-12-09.
+* Michael: There will be support to finalize the errata (technical writer)
+* David: Checklist for finalizing errata is in github issue [Requirements to finalize the errata #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+## 3.3 Review current state of ecosystem ongoing work
+
+* Eddy:
+  - Github capabilities for SARIF extensions for VS and VSCode
+  - Improve Codeflows for tool (CodeQL)
+  - New ITM version
```
Comment (eddynaka) [inline meeting_minutes/220113_SARIF_TC_50.md:101]:
```suggestion
  - New RTM version (RTM 6)
```

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+* Aditya: Offers support for the protocol to avoid duplication of efforts
+* All discuss options for a result store
+* Nathan: How does the envisaged result store handle static and dynamic analysis results
+* All discuss options for including dynamic results in a result store and to access per CIP 
+* Nathan: Shortly lists the typical activities and result types that his team performs and uses
+* David: Suggests to maybe consider eclipse to provide diversity of extensions
+* Paul: Suggests to be careful to decide on what "flavor" to target (as many vendors provide customized versions)
+* Stefan: Proposes to approach JetBrains - as they e.g. offer [Qodana - a linter with SARIF output](https://www.jetbrains.com/help/qodana/qodana-sarif-output.html) 
+* James: Maybe can present some of the Snyk tools in one of the next meetings
+* James: Consuming SARIF files can be resoruce intensive in the current version due to the format chosen - are there plans to change that or how do other tools handle that?
```
Comment (eddynaka) [inline meeting_minutes/220113_SARIF_TC_50.md:131]:
```suggestion
* James: Consuming SARIF files can be resource intensive in the current version due to the format chosen - are there plans to change that or how do other tools handle that?
```

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+* ACTION Michael: Reach out to ParaSoft
+  - ONGOING
+* ACTION Michael: Will author and check in the roadmap / list of in- and out-of-scope features
+  - ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* James Spoor first time participation amd if he attends the next meeting will gain voting rights at the end of that meeting.
```
Comment (katrinaoneil) [inline meeting_minutes/220113_SARIF_TC_50.md:49]: amd -> and

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Aditya Sharad and Nathan Baird both will gain voting rights at the end of next meeting.
```
Comment (katrinaoneil) [inline meeting_minutes/220113_SARIF_TC_50.md:53]: next -> this

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+* ONGOING otherwise no updates from the committee
+
+## 3.5 Review proposal for what's in- and out-of-scope - general discussion
+
+* ONGOING and tracked by Michael's action on the metrics / roadmap
+
+## 3.6 Status of Wikipedia page
+
+* ONGOING and David plans to provide until one of the two next meetings
```
Comment (katrinaoneil) [inline meeting_minutes/220113_SARIF_TC_50.md:114]: not sure I understand...

Comment (sthagen) [reply meeting_minutes/220113_SARIF_TC_50.md:114]: @dmk42: I meant provide as synonym to publish,  but maybe best you revisit and state your intent more clearly than I achieved?

Comment (katrinaoneil) [reply meeting_minutes/220113_SARIF_TC_50.md:114]: Ah, got it!

**Review by katrinaoneil — APPROVED** [review]

**Review by dmk42 — APPROVED** [review]
LGTM

File: meeting_minutes/220113_SARIF_TC_50.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+| :--------- | :-------- | :--------------- | :---------------------- |
+| Aditya     | Sharad    | Microsoft        | Member                  |
+| Courtney   | Lawton    | Microsoft        | Public Participant      |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| James      | Spoor     | Snyk Ltd         | Member                  |
+| Katrina    | O'Neil    | Micro Focus      | Voting member           |
+| Nathan     | Baird     | Microsoft        | Member                  |
+| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
+| Stefan     | Hagen     | Individual       | Secretary, taking notes |
```
Comment (katrinaoneil) [inline meeting_minutes/220113_SARIF_TC_50.md:17]: just noticed: wasn't Michael present at the meeting as well?

Comment (dmk42) [reply meeting_minutes/220113_SARIF_TC_50.md:17]: Yes, he was.  Sorry I missed this.  This is a correction we should mention during approval of the minutes.

**Pull request merged**